casper/src/test/scala/coop/rchain/casper/genesis/GenesisTest.scala

```diff
@@ -299,6 +299,7 @@ object GenesisTest {
                            numberOfActiveValidators = numberOfActiveValidators,
                            validators = validators,
                            posMultiSigPublicKeys = GenesisBuilder.defaultPosMultiSigPublicKeys,
+                           posVaultPubKey = GenesisBuilder.defaultPosVaultPubKey,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.